types/miromico/REV2_V5/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "FMLR IoT Button V2.5.0",
+  "name": "miro Click V2.5.0",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed